app/views/users/registrations/edit.html.erb

```diff
@@ -28,7 +28,7 @@
 
     <h3>Cancel my account</h3>
 
-    <p>Unhappy? <%= link_to "Cancel my account", registration_path(resource_name), data: { confirm: "Are you sure? We will delete your personal information however we keep your contributions and we assign them to another anonymous account." }, method: :delete %></p>
+    <p>Unhappy? <%= link_to "Cancel my account", registration_path(resource_name), data: { confirm: "Are you sure? We will cancel your access to the website however your contributions will still be displayed, a little bit like for the same reasons than Wikipedia." }, method: :delete %></p>
```

Comment:
'the same reasons than' -> 'the same reasons as'
isn't 'a bit like for' an anacoluthon? maybe we can just say:
> Are you sure? We will cancel your access to the website, however your contributions will still be displayed, a little bit like Wikipedia's account deactivation process.

and then maybe add a link to https://en.wikipedia.org/wiki/Wikipedia:FAQ#How_do_I_change_my_username/delete_my_account?

Reply:
Yep, sure!